onnxruntime/test/platform/apple/apple_package_test/Podfile.template

```diff
@@ -1,14 +1,34 @@
-platform :ios, '13.0'
+def should_exclude_macos_target
+    if '@C_POD_NAME@' != 'onnxruntime-c'
```

Comment:
currently only `onnxruntime-mobile-c` cannot run on MacOS right?
```suggestion
    if '@C_POD_NAME@' == 'onnxruntime-mobile-c'
```

Reply:
I think currently I only enabled macos arch in the default_apple_json files, so training/mobile variant both don't run macos yet.